### ISSUE TITLE
cpu_time always 0.0

### ISSUE THREAD

**Issue by LeonardSchmiester** [ICB-DCM/AMICI#1124]
I simulated all benchmark models with the `simulate_petab` function (develop branch from today, but also happens for the current master branch) and the `cpu_time` in the returned `rdatas` is `0.0` for all models.

I tested it with forward / adjoint / no sensis and always got the same result for the cpu time.

```
petab_problem = petab.Problem.from_yaml(f'{benchmark_model}.yaml')

sys.path.insert(0, benchmark_model)
model_module = importlib.import_module(benchmark_model)
amici_model = model_module.getModel()

amici_solver = amici_model.getSolver()
amici_solver.setSensitivityOrder(1)

sim_results = simulate_petab(petab_problem=petab_problem,
                             amici_model=amici_model,
                             solver=amici_solver)

sim_results['rdatas'][0]['cpu_time']
Out[13]: 0.0
```

**Comment by dweindl** [comment]
Problem is that `ReturnData::processSolver` isn't called anywhere.

**Comment by dweindl** [comment]
Also all of numSteps, numrhsevals, numnonlinsolvconvfails are probably empty / all zero.

**Comment by LeonardSchmiester** [comment]
> Also all of numSteps, numrhsevals, numnonlinsolvconvfails are probably empty / all zero.

Yes, can confirm that they are all zero.

**Comment by dweindl** [comment]
Proposed fix in 24d3497

**Comment by LeonardSchmiester** [comment]
:+1: that fixed the issue. Thanks
